lib/material_button/material_button.dart

```diff
@@ -21,7 +23,7 @@ import 'material_button_base.dart';
 /// </material-button>
 /// ```
 ///
-/// ## Attributes
+/// __Attributes:__
```

Comment:
**Question**. What is double underscore format, and why the change from a heading?

Reply:
My guess would be to make the styling more consistent on the button page of the gallery:
https://dart-lang.github.io/angular_components_example/#/material_button